client/reader/tag-stream/style.scss

```diff
@@ -14,6 +14,7 @@
 }
 
 .tag-stream__header .follow-button {
+	z-index: 1;
```

Comment:
It's probably better to use the scss z-index function here: https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md#z-index